### ISSUE TITLE
Training, testing methodology

### ISSUE THREAD

**Issue by marmundo** [marmundo/TouchAnalytics#9]
Supose 41 users.

To train a classifier with 21 users and test it with the remaining 20 users.
*Guarantee that the user samples in test set is not in training set*

Create a score matrix based on this test approach.
